src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -78,6 +78,8 @@ public function save(Key $key)
             }
         }
 
+        $this->checkNotExpired($key);
```

Comment:
this check is duplicated in `Store`s and `Lock`. The goal was to be sure that the check is performed even when user call the store without using our `Lock` facade.
But I wonder if it worth it: this leads to duplicated code and some stores forget to implement it (ie. ZooKeeper)